cmd/limactl/start-at-login.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"runtime"
+
+	"github.com/lima-vm/lima/pkg/autostart"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func startAtLoginCommand() *cobra.Command {
+	resetCommand := &cobra.Command{
+		Use:               "start-at-login INSTANCE",
+		Short:             "Register/Unregister the unit file for instance",
+		Args:              WrapArgsError(cobra.MaximumNArgs(1)),
+		RunE:              startAtLoginAction,
+		ValidArgsFunction: startAtLoginComplete,
+		GroupID:           advancedCommand,
+	}
+
+	resetCommand.Flags().Bool(
+		"enabled", true,
+		"--enabled=true/false the "+map[string]string{"darwin": "launchd", "linux": "systemd"}[runtime.GOOS]+" unit file for instance",
```

Comment:
```suggestion
		"the "+map[string]string{"darwin": "launchd", "linux": "systemd"}[runtime.GOOS]+" unit file for instance",
```

Reply:
The flag shouldn't be added for Windows, NetBSD, etc

Reply:
Flag will be [added only on linux or darwin](https://github.com/lima-vm/lima/pull/2151/commits/d66089e94c84446a04b0d7498139c801123ec1aa#diff-95475f7f5560ba438d76bce7e63eb2f2d63b59438023d2af973e941510cdcf9cR126)